autogpt_platform/frontend/src/components/ui/checkbox.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Checkbox } from "./checkbox";
+
+const meta = {
+  title: "UI/Checkbox",
+  component: Checkbox,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+  argTypes: {
+    checked: {
+      control: "boolean",
+    },
+    disabled: {
+      control: "boolean",
+    },
+  },
+} satisfies Meta<typeof Checkbox>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {},
+};
+
+export const Checked: Story = {
+  args: {
+    checked: true,
+  },
+};
+
+export const Unchecked: Story = {
+  args: {
+    checked: false,
+  },
+};
+
+export const Disabled: Story = {
+  args: {
+    disabled: true,
+  },
+};
+
+export const DisabledChecked: Story = {
+  args: {
+    disabled: true,
+    checked: true,
+  },
+};
+
+export const WithLabel: Story = {
+  args: {},
+  render: (args) => (
+    <div className="flex items-center space-x-2">
+      <Checkbox id="terms" {...args} />
+      <label
```

Comment:
we have a `Label` component

Reply:
This is testing the checkbox component, it's best to use plain HTML for anything else in the story. Just like normal tests you don't want to be needing to pull in other classes that you are not testing